pkg/postgres/schema/process_indicators.go

```diff
@@ -47,16 +50,20 @@ const (
 )
 
 // ProcessIndicators holds the Gorm model for Postgres table `process_indicators`.
+// XXX: ROX-13790 unique_indicator index is specified manually, as it has to
+// cover multiple columns, which is not supported by the store generator.
+//
+// Note, that it will introduce some overhead even with PLOP feature disabled.
 type ProcessIndicators struct {
 	Id                 string `gorm:"column:id;type:uuid;primaryKey"`
 	DeploymentId       string `gorm:"column:deploymentid;type:uuid;index:processindicators_deploymentid,type:hash"`
-	ContainerName      string `gorm:"column:containername;type:varchar"`
-	PodId              string `gorm:"column:podid;type:varchar"`
+	ContainerName      string `gorm:"column:containername;type:varchar;index:unique_indicator"`
```

Comment:
Could the index name be prefixed similarly to the `processindicators_poduid` one ?

Reply:
Sure, makes sense.